src/main/kotlin/se/uulm/snowballr/backend/service/ProjectService.kt

```diff
@@ -194,4 +207,76 @@ class ProjectService(
             val projectMembersWithUsers = projectMemberRepo.getProjectMembersWithUsers(projectId)
             projectMembersWithUsers.toGrpcProjectMembers()
         }
+
+    override suspend fun getDecisionStatisticsForStage(
+        request: GrpcProjectDecisionStatistics.Get,
+    ): GrpcProjectDecisionStatistics = withUser(userRepo) { currentUser ->
+        val projectId = parseUUID(request.projectId, EntityType.PROJECT)
+
+        isAllowedToReadProject(projectMemberRepo).checkFor(currentUser, projectId)
+
+        val project = repo.getProjectById(projectId).getOrThrow()
+        val currentStage = project.currentStage
+
+        if (request.stage > currentStage) {
+            throw StageNotFoundException(request.stage)
+        }
+
+        val statistics = createStatistics(projectId, request.stage)
+        GrpcProjectDecisionStatistics
+            .newBuilder()
+            .addAllStatistics(statistics)
+            .build()
+    }
+
+    /**
+     * Creates a list of [GrpcProjectDecisionStatistics.Statistic] objects for the given [projectId] and [stage].
+     *
+     * @param projectId The ID of the project for which the statistics should be created.
+     * @param stage The stage for which the statistics should be created.
+     * @return A list of [GrpcProjectDecisionStatistics.Statistic] objects.
+     */
+    private suspend fun createStatistics(projectId: UUID, stage: Long): List<GrpcProjectDecisionStatistics.Statistic> {
+        var acceptedCount = 0L
+        var declinedCount = 0L
+        var unreviewedCount = 0L
+        var inReviewCount = 0L
+
+        projectPaperRepo.getAllProjectPapersForProject(projectId).forEach { projectPaper ->
+            if (projectPaper.stage == stage) {
+                when (projectPaper.decision) {
+                    ProjectOuterClass.PaperDecision.PAPER_DECISION_ACCEPTED -> acceptedCount++
```

Comment:
```suggestion
                    PaperDecision.PAPER_DECISION_ACCEPTED -> acceptedCount++
```
If you import the `PaperDecision` directly, would reduce the line length a bit.